addusers.py

```diff
@@ -357,6 +360,29 @@ def parse_rows(rows, select_user=None):
     auth_url = config.get('auth', 'auth_url')
     nova_version = config.get('nova', 'version')
 
+    approved_column = config.get('header_column', 'approved_column')
```

Comment:
1) we don't use most of these, we should only have config values for headers we actually care about.  You never want to have variables laying around that don't get used.
  
2) But, if you ever do need to load a whole config section, you can do it much more cleanly like this:
`columns = config.items('header_column')`
This creates a list of tuples, you can use them that way or put them into a dictionary with `dict(columns)`